lib/ruby_saml/settings.rb

```diff
@@ -438,5 +443,11 @@ def get_sp_certs_multi
       certs.each { |_, ary| ary.freeze }
       certs
     end
+
+    def build_cert_object(cert)
+      return cert if cert.is_a?(OpenSSL::X509::Certificate)
+
+      OneLogin::RubySaml::Utils.build_cert_object(cert)
```

Comment:
let's just change this method in OneLogin::RubySaml::Utils to return a cert if one is given as an argument (it should be "idempotent")

Reply:
👍 Your code, your decision 😄 Will change that.